recipes/pdoc3/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "pdoc3" %}
+{% set version = "0.7.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "df43a7f1a139a5a61e778f424f167719acc33ed71bf13b6c34c7ebd139866eb7"
+
+build:
+  number: 0
+  noarch: python
+  entry_points:
+    - pdoc = pdoc.cli:main
+    - pdoc3 = pdoc.cli:main
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - mako
+    - markdown >=3.0
+    - pip
+    - python >=3.5
+    - setuptools_scm
+    - setuptools-git
+  run:
+    - mako
+    - markdown >=3.0
+    - python >=3.5
+
+test:
+  imports:
+    - pdoc
+    - pdoc.test
+    - pdoc.test.example_pkg
```

Comment:
Missing are some API modules such as `pdoc.cli`, `pdoc.html_helpers`.

Reply:
Thanks, will add.  The import list need not be exhaustive - enough to cover basic bits, and especially any compiled extensions.